<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>Form</class>
    <widget class="QWidget" name="Form">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>599</width>
                <height>463</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>替换值</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
                <widget class="QFrame" name="frame">
                    <property name="frameShape">
                        <enum>QFrame::StyledPanel</enum>
                    </property>
                    <property name="frameShadow">
                        <enum>QFrame::Raised</enum>
                    </property>
                    <layout class="QVBoxLayout" name="verticalLayout">
                        <item>
                            <widget class="QLabel" name="label">
                                <property name="text">
                                    <string>可以多选</string>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <widget class="QListWidget" name="column_widget">
                                <property name="selectionMode">
                                    <enum>QAbstractItemView::ExtendedSelection</enum>
                                </property>
                            </widget>
                        </item>
                    </layout>
                </widget>
            </item>
            <item>
                <widget class="QFrame" name="frame_2">
                    <property name="frameShape">
                        <enum>QFrame::StyledPanel</enum>
                    </property>
                    <property name="frameShadow">
                        <enum>QFrame::Raised</enum>
                    </property>
                    <layout class="QVBoxLayout" name="verticalLayout_3">
                        <item>
                            <widget class="QTabWidget" name="tabwidget">
                                <property name="currentIndex">
                                    <number>0</number>
                                </property>
                                <widget class="QWidget" name="tab">
                                    <attribute name="title">
                                        <string>方式1</string>
                                    </attribute>
                                    <layout class="QVBoxLayout" name="verticalLayout_2">
                                        <item>
                                            <widget class="QFrame" name="frame_5">
                                                <property name="frameShape">
                                                    <enum>QFrame::StyledPanel</enum>
                                                </property>
                                                <property name="frameShadow">
                                                    <enum>QFrame::Raised</enum>
                                                </property>
                                                <layout class="QHBoxLayout" name="horizontalLayout_2">
                                                    <item>
                                                        <widget class="QLabel" name="label_3">
                                                            <property name="text">
                                                                <string>原值</string>
                                                            </property>
                                                        </widget>
                                                    </item>
                                                    <item>
                                                        <widget class="QLineEdit" name="old_lineEdit">
                                                            <property name="text">
                                                                <string>,</string>
                                                            </property>
                                                        </widget>
                                                    </item>
                                                </layout>
                                            </widget>
                                        </item>
                                        <item>
                                            <widget class="QFrame" name="frame_6">
                                                <property name="frameShape">
                                                    <enum>QFrame::StyledPanel</enum>
                                                </property>
                                                <property name="frameShadow">
                                                    <enum>QFrame::Raised</enum>
                                                </property>
                                                <layout class="QHBoxLayout" name="horizontalLayout_4">
                                                    <item>
                                                        <widget class="QLabel" name="label_4">
                                                            <property name="text">
                                                                <string>新值</string>
                                                            </property>
                                                        </widget>
                                                    </item>
                                                    <item>
                                                        <widget class="QLineEdit" name="new_lineEdit">
                                                            <property name="text">
                                                                <string>;</string>
                                                            </property>
                                                        </widget>
                                                    </item>
                                                </layout>
                                            </widget>
                                        </item>
                                    </layout>
                                </widget>
                                <widget class="QWidget" name="tab_2">
                                    <attribute name="title">
                                        <string>方式2</string>
                                    </attribute>
                                    <layout class="QVBoxLayout" name="verticalLayout_4">
                                        <item>
                                            <widget class="QFrame" name="frame_4">
                                                <property name="frameShape">
                                                    <enum>QFrame::StyledPanel</enum>
                                                </property>
                                                <property name="frameShadow">
                                                    <enum>QFrame::Raised</enum>
                                                </property>
                                                <layout class="QHBoxLayout" name="horizontalLayout_3">
                                                    <item>
                                                        <widget class="QLabel" name="label_2">
                                                            <property name="text">
                                                                <string>字符</string>
                                                            </property>
                                                        </widget>
                                                    </item>
                                                    <item>
                                                        <widget class="QLineEdit" name="other_linedit"/>
                                                    </item>
                                                </layout>
                                            </widget>
                                        </item>
                                        <item>
                                            <widget class="QFrame" name="frame_7">
                                                <property name="frameShape">
                                                    <enum>QFrame::StyledPanel</enum>
                                                </property>
                                                <property name="frameShadow">
                                                    <enum>QFrame::Raised</enum>
                                                </property>
                                                <layout class="QHBoxLayout" name="horizontalLayout_5">
                                                    <item>
                                                        <widget class="QRadioButton" name="rbt00">
                                                            <property name="text">
                                                                <string>保留</string>
                                                            </property>
                                                            <property name="checked">
                                                                <bool>true</bool>
                                                            </property>
                                                        </widget>
                                                    </item>
                                                    <item>
                                                        <widget class="QRadioButton" name="rbt11">
                                                            <property name="text">
                                                                <string>舍弃</string>
                                                            </property>
                                                        </widget>
                                                    </item>
                                                </layout>
                                            </widget>
                                        </item>
                                    </layout>
                                </widget>
                            </widget>
                        </item>
                        <item>
                            <widget class="QFrame" name="frame_3">
                                <property name="frameShape">
                                    <enum>QFrame::StyledPanel</enum>
                                </property>
                                <property name="frameShadow">
                                    <enum>QFrame::Raised</enum>
                                </property>
                                <layout class="QHBoxLayout" name="horizontalLayout">
                                    <item>
                                        <widget class="QRadioButton" name="rbt0">
                                            <property name="text">
                                                <string>替换当前值</string>
                                            </property>
                                            <property name="checked">
                                                <bool>true</bool>
                                            </property>
                                            <attribute name="buttonGroup">
                                                <string notr="true">buttonGroup</string>
                                            </attribute>
                                        </widget>
                                    </item>
                                    <item>
                                        <widget class="QRadioButton" name="rbt1">
                                            <property name="text">
                                                <string>添加新列</string>
                                            </property>
                                            <attribute name="buttonGroup">
                                                <string notr="true">buttonGroup</string>
                                            </attribute>
                                        </widget>
                                    </item>
                                </layout>
                            </widget>
                        </item>
                        <item>
                            <widget class="QPushButton" name="ok_button">
                                <property name="styleSheet">
                                    <string notr="true">color: rgb(255, 255, 255); background-color: rgb(28, 177,
                                        245);
                                    </string>
                                </property>
                                <property name="text">
                                    <string>OK</string>
                                </property>
                            </widget>
                        </item>
                    </layout>
                </widget>
            </item>
        </layout>
    </widget>
    <resources/>
    <connections/>
    <buttongroups>
        <buttongroup name="buttonGroup"/>
    </buttongroups>
</ui>
